core/ui/ListItemTemplate.tid

```diff
@@ -1,7 +1,5 @@
 title: $:/core/ui/ListItemTemplate
 
 <div class="tc-menu-list-item">
-<$link to={{!!title}}>
-<$view field="title"/>
-</$link>
+<$link to={{!!title}}><$view field="title"/></$link>
```

Comment:
Is there a reason to not merely use <$link/> ? (Or is perhaps !!title not always same as currentTiddler?)

Reply:
hmm, should be faster too. Will do some testing. thx

Reply:
@twMat's point is worth addressing I think

Reply:
I'll add it. I did some testing already and everything seemed to be OK.